pkg/appconsts/v1/app_consts.go

```diff
@@ -6,10 +6,16 @@ const (
 	Version              uint64 = 1
 	SquareSizeUpperBound int    = 128
 	SubtreeRootThreshold int    = 64
-	TimeoutPropose              = time.Second * 10
-	TimeoutCommit               = time.Second * 11
+	// Note: TimeoutPropose and TimeoutCommit weren't actually constants in v1,
+	// they were defaults for user-configurable timeouts. TODO: consider
```

Comment:
apparently it's used in the v1 -> v2 tests:

```
func upgradeFromV1ToV2(t *testing.T, testApp *app.App) {
	t.Helper()
	testApp.BeginBlock(abci.RequestBeginBlock{Header: tmproto.Header{
		Height:  2,
		Version: tmversion.Consensus{App: 1},
	}})
	endBlockResp := testApp.EndBlock(abci.RequestEndBlock{Height: 2})
	require.Equal(t, appconsts.GetTimeoutCommit(v1.Version),
		endBlockResp.Timeouts.TimeoutCommit)
	require.Equal(t, appconsts.GetTimeoutPropose(v1.Version),
		endBlockResp.Timeouts.TimeoutPropose)
	testApp.Commit()
	require.EqualValues(t, 2, testApp.AppVersion())
}
```

so it's fine to leave, I guess

Reply:
Thanks! I'll remove the TODO.